js/index.js

```diff
@@ -1,7 +1,106 @@
+console.log(hacker1);
+
+/* 3.2 Print all the characters of the navigator's name, in reverse order. 
+  i.e. `"nhoJ"`*/
+
+for (let i = (hacker1.length); i >= 0; i-- ){
+    console.log(hacker1.charAt(i))
+}
+
+/*3.3 Depending on the [lexicographic order](https://en.wikipedia.org/wiki/Lexicographical_order) of the strings, print: <br>
+    - `The driver's name goes first.` <br>
+    - `Yo, the navigator goes first definitely.` <br>
+    - `What?! You both have the same name?`*/
+
+switch(hacker1.localeCompare(hacker2)) {
+    case 1: console.log(`The driver's name goes first.`);
+    break;
+    case -1: console.log(`Yo, the navigator goes first definitely.`)
+    break;
+    case 0: console.log(`What?! You both have the same name?`)
+    break;
+  
+  }
+
+//#### Bonus 1:
+//Go to [lorem ipsum generator](http://www.lipsum.com/) and:
+//Generate 3 paragraphs. Store the text in a variable type of string.
+//Make your program count the number of words in the string.
+
+  
+let LoremStr = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras bibendum velit eget eros blandit, eu vestibulum libero rutrum. Etiam in vestibulum enim. Nunc eget eros quis ipsum porttitor sollicitudin. Fusce porta libero erat, vel faucibus ligula rutrum nec. Quisque vitae enim scelerisque, blandit nunc vitae, suscipit erat. In mi libero, eleifend et euismod quis, vehicula eget mi. Aenean nec libero ut libero euismod convallis. 
+
+Quisque sed velit nec lorem vulputate volutpat non id massa. Aenean sed diam egestas ipsum dignissim facilisis eget eget libero. Etiam non erat massa. Proin bibendum, augue at fermentum finibus, metus odio molestie lectus, eu vulputate ipsum erat viverra tellus. Nam euismod vitae dolor sit amet facilisis. Sed nisi turpis, fringilla sed mi sed, cursus facilisis lectus. Suspendisse ut justo faucibus, faucibus sem eget, congue orci. Maecenas lobortis turpis libero, id venenatis elit porta pulvinar. Vivamus id ipsum id lorem vestibulum pulvinar. Nunc a magna placerat, viverra ex nec, mollis ante. Praesent pellentesque lectus at orci congue, nec viverra est suscipit. Duis in dictum augue. Aenean tincidunt luctus nunc eget tempus. 
+
+Sed laoreet ante vel quam auctor efficitur. Curabitur non metus vitae quam sollicitudin aliquet. Nunc imperdiet mi tortor, maximus sollicitudin urna volutpat et. Nunc eget nunc malesuada, ultrices tellus et, mollis orci. Pellentesque dapibus diam felis, sit amet consectetur lacus mollis at. Nunc gravida purus eget nibh bibendum mollis. Nulla ultricies metus ac turpis laoreet posuere. Proin eu dolor vel augue venenatis congue. Pellentesque convallis lorem id odio mattis, non sodales leo hendrerit. Nunc at malesuada metus.`
+
+
+function WordCount(str) { 
+  return str.split(" ").length;
+}
+
+console.log(WordCount(LoremStr));
+
+// Make your program count the number of times the Latin word [`et`] appears.
+
+function WordEtCount(str) { 
+    return str.split(`et`).length;
```

Comment:
Nice! Question: does this solution look for the word or the substring "et"? ("Make your program count the number of times the Latin word et appears")

Reply:
good question, it was difficult to figure. At this point I think it looks for just _'et'_ so it could also be inside a word (substring).
If you look for _' et '_ (with spaces) you could also be missing sentences that end with **et.**

Reply:
I added the sapces to make sure it only searches for **et**, but it might break if **et** is at the end of a sentence. (which would be weird since **et** means and, that i'm aware of)